shortuuid/django_fields.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             kwargs["max_length"] = self.length + len(self.prefix)  # type: ignore
 
         self.alphabet: str = kwargs.pop("alphabet", None)  # type: ignore
-        kwargs["default"] = self._generate_uuid  # type: ignore
+        kwargs["default"] = kwargs["default"] if "default" in kwargs else self._generate_uuid  # type: ignore
```

Comment:
Isn't this equivalent to:

```suggestion
        kwargs["default"] = kwargs.get("default", self._generate_uuid)
```

?